package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "joi-image-extensions",
+  "version": "0.1.0",
+  "description": "Joi image validation extension",
+  "main": "lib/index.js",
+  "keywords": [
+    "joi",
+    "validation",
+    "extension",
+    "image",
+    "dimensions",
+    "size"
+  ],
+  "scripts": {
+    "browsersync:coverage": "browser-sync start --port 1337 --startPath 'lcov-report' --server coverage --files 'coverage/**/*.*'",
+    "build": "rimraf ./lib && babel ./src --out-dir lib",
+    "coverage": "NODE_ENV=test multi='spec=- mocha-junit-reporter=-' nyc --reporter=lcov mocha --reporter mocha-multi",
+    "check-coverage": "nyc report --reporter=text-summary && nyc check-coverage",
+    "lint": "eslint .",
+    "mocha": "NODE_ENV=test mocha",
+    "prepare": "npm run build",
+    "test": "npm run -s lint && npm run -s coverage && npm run -s check-coverage",
+    "watch:build": "nodemon --exec 'npm run -s build' -w src",
+    "watch:coverage": "nodemon --exec 'npm run -s coverage && npm run -s check-coverage' -w src -w test",
+    "watch:lint": "nodemon --exec 'npm run -s lint' -w src -w test",
+    "watch:mocha": "nodemon --exec 'npm run -s mocha' -w src -w test",
+    "watch:test": "nodemon --exec 'npm run -s test' -w src -w test"
+  },
+  "author": "chris@doing.io",
+  "license": "ISC",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/doinginc/joi-image-extension.git"
+  },
+  "license": "UNLICENSED",
```

Comment:
I'd personally go with an MIT license, though ultimately we should get @technojosh's input since its a legal question. Though it likely doesn't make sense for this package since it's associated with Joi, any future work that may produce trademarks worth protecting warrants considering a BSD 3/4-clause.